packages/cli/src/lib/bundler/config.ts

```diff
@@ -347,6 +347,13 @@ export async function createConfig(
   return {
     mode,
     profile: false,
+    ...(isDev
+      ? {
+          watchOptions: {
+            ignored: /node_modules\/(?!\@backstage)/,
```

Comment:
`@backstage` is a magic string here, what is it that it's actually trying to express? Is it to still keep file watching on the `@backstage` packages specifically or is this more of a thing for the main repo? Is it needed at all of could we leave it out

Reply:
I kept that magic string because it was part of the configuration in `createBackendConfig`.  However, removing it made no difference when I tested with both node_modules and PnP.  We can leave it out, if you'd prefer that approach.

Just let me know and I'll remove it from the PR.

Reply:
I went ahead and removed it, since it wasn't necessary for the webpack functionality to work with Yarn PnP.